### ISSUE TITLE
Linux snaps have wrong build number

### ISSUE THREAD

**Issue by craxal** [microsoft/AzureStorageExplorer#1528]
**Storage Explorer Version**: 1.9.0
**Build Number**: 20190627.4
**Branch**: master
**Platform/OS**: Linux Ubuntu

## Bug Description ##
Installed Linux snaps don't appear to report the correct build number as the build they come from.

## Steps to Reproduce ##
1. Download the Linux snap artifact for build 20190627.4
2. Install snap and connect password manager service.
3. Launch Storage Explorer and open the About dialog.

## Expected Experience ##
The build number is 20190627.4

## Actual Experience ##
The build number is 20190627.3

## Additional Context ##
![](https://user-images.githubusercontent.com/34729022/60145616-30081e00-97f9-11e9-8188-326b53d7b967.png)

**Comment by craxal** [comment]
@v-xuanzh Can you also check the commit ID against the build?

I think I know what the problem is:

Snap builds use `multipass`, which spins up a VM wherein `snapcraft` does it's work, including copying the .tar.gz we generate in our packaging step. The VM, however, is not cleaned or deleted when the snap is finished. I imagine this is to make subsequent snap builds much faster, because then `snapcraft` doesn't have to pull and build sources every time.

But herein is the problem. Since a .tar.gz file is already present from the first build, `snapcraft` thinks it doesn't need to pull the source anymore, so it uses the one already there. That's why the Storage Explorer snap build number doesn't match the one in the .tar.gz.

**Comment by v-xuanzh** [comment]
Hi @craxal ,We verified this issue on the latest master build 20190704.5. It has been fixed for now.

But I found a new issue.
The shown version of the installed Storage Explorer is 1.8.1 using the command `sudo snap install StorageExplorer_latest_amd64.snap --dangerous`
![image](https://user-images.githubusercontent.com/34729022/60692504-e27e6600-9f08-11e9-8ef9-6a713c104ff1.png)

Actually, it should be 1.9.0
